### PR TITLE
harfbuzz: 1.1.0

### PR DESCRIPTION
**Pull request by moyogo**
with patch from https://github.com/behdad/harfbuzz/issues/183

### CODE REVIEW

File: Library/Formula/harfbuzz.rb [hunk 1]
```diff
@@ -34,6 +34,10 @@ class Harfbuzz < Formula
     sha256 "9535d35dab9e002963eef56757c46881f6b3d3b27db24eefcc80929781856c77"
   end
 
+  def patches
```
Comment (mikemcquaid) [inline Library/Formula/harfbuzz.rb]: Use `patch do` instead.

File: Library/Formula/harfbuzz.rb [hunk 2]
```diff
@@ -34,6 +34,10 @@ class Harfbuzz < Formula
...
+    DATA
```
Comment (mikemcquaid) [inline Library/Formula/harfbuzz.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

Comment (moyogo) [inline Library/Formula/harfbuzz.rb]: Is it OK if the comment points to the issue upstream that has a patch in a comment?
```ruby
  # See https://github.com/behdad/harfbuzz/issues/183
  patch do
    DATA
  end
```

Comment (mikemcquaid) [inline Library/Formula/harfbuzz.rb]: Given it's a GitHub repository: submitting a pull request would be good.

Comment (moyogo) [inline Library/Formula/harfbuzz.rb]: Ah I see, `patch do` is really cool. I'll use that.

**Comment by moyogo** [comment]
@mikemcquaid There’s now a pull request upstream behdad/harfbuzz#184 and `patch do` is used.

**Comment by moyogo** [comment]
Closing since harfbuzz 1.1.1 just got released.

**Pull request closed without merging**